samples/client/petstore/swift3/default/PetstoreClient/Classes/Swaggers/APIHelper.swift

```diff
@@ -8,27 +8,25 @@ import Foundation
 
 class APIHelper {
     static func rejectNil(_ source: [String:Any?]) -> [String:Any]? {
-        var destination = [String:Any]()
-        for (key, nillableValue) in source {
-            if let value: Any = nillableValue {
-                destination[key] = value
-            }
-        }
-
-        if destination.isEmpty {
-            return nil
+        let result = source.reduce([String:Any]()) { (dict, tuple:(key: String, value: Any?)) -> [String:Any] in
+            guard let value = tuple.value else { return dict }
+            var dict = dict
+            dict.updateValue(value, forKey: tuple.key)
+            return dict
         }
-        return destination
+        
+        if result.isEmpty { return nil }
+        return result
```

Comment:
@wolfAle I would suggest removing this particular change from this PR so that we can merge this PR into master. 

Then you can file another PR with this particular change to continue the discussion/debate.